src/components/ResourceInstance/Connectivity/ConnectivityEndpoint.tsx

```
@@ -48,25 +48,31 @@ const ResourceConnectivityEndpoint: FC<ResourceConnectivityEndpointProps> = (
+    const portValues =
+      typeof selectedPortItem?.ports === "string"
+        ? selectedPortItem.ports.split(",").map((port) => Number(port.trim()))
+        : Array.isArray(selectedPortItem?.ports)
+          ? selectedPortItem.ports.map((port) =>
+              typeof port === "string" ? Number(port.trim()) : port
+            )
+          : typeof ports === "string"
+            ? ports.split(",").map((port) => Number(port.trim()))
+            : [];
+
+    return portValues.filter((port) => !isNaN(port));
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)